packages/logo-grid/index.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useRef, useState } from 'react'
+import Image from '@hashicorp/react-image'
+import Button from '@hashicorp/react-button'
+import fragment from './fragment.graphql'
+import classNames from 'classnames'
+import DialogTooltip from './partials/dialog-tooltip'
+import { useRect } from '@reach/rect'
+import s from './style.module.css'
+
+interface CompanyLogo {
+  /** url of the image */
+  url: string
+  /** format of the image, like "jpg" or "svg" */
+  format: string
+  /** alt text for the image. defaults to company name. */
+  alt?: string
+}
+
+interface Company {
+  /** Company name */
+  name: string
+  /** Short description of what the company offers */
+  description: string
+  /** Optional integration page to link to. */
+  integrationPage?: {
+    /** Slug to link to, to be constructed into a `.com/integrations/{slug}` url. */
+    slug: string
+  }
+  /** Company website address. */
+  link?: string
+  /** Color logo image. */
+  logo?: CompanyLogo
+  /** Monochrome black logo image. */
+  monochromeLogo?: CompanyLogo
+  /** Monochrome white logo image. */
+  whiteLogo?: CompanyLogo
+}
+
+interface LogoGridProps {
+  /** Array of company objects to render as grid items. */
+  data: Array<Company>
```

Comment:
nit: The more conventional way to define an array type is:
```suggestion
  data: Company[]
```